### PR TITLE
Add ability to get list of users with profile

### PR DESCRIPTION
**Pull request by gfedorenko**

### CODE REVIEW

**Review by ec — APPROVED** [review]

File: spec/api/v2/admin/users_spec.rb [hunk 1]
```diff
@@ -55,6 +56,26 @@ def validate_fields(user)
...
+      end
+      let(:do_extended_info_request) { get '/api/v2/admin/users', headers: auth_header, params: extended_params }
+
+      it 'returns list of users with full info' do
+        do_extended_info_request
+        users = JSON.parse(response.body)
+        expect(User.count).to eq users.count
+
+        expect(users.first['labels']).to eq []
+        expect(users.first['activities']).to eq []
```
Comment (calj) [inline spec/api/v2/admin/users_spec.rb]: We need remove activities from this call.

Comment (gfedorenko) [reply spec/api/v2/admin/users_spec.rb]: @calj  So we should remove them in entity?

File: spec/api/v2/admin/users_spec.rb [hunk 2]
```diff
@@ -41,6 +41,7 @@
       def validate_fields(user)
         user.attributes.slice('email', 'role', 'level', 'otp', 'state', 'uid')
       end
+
       it 'returns list of users' do
         do_request
         users = JSON.parse(response.body)
```
Comment (calj) [inline spec/api/v2/admin/users_spec.rb]: Assert here that extended informations are missing.

File: spec/api/v2/admin/users_spec.rb (hunk 1 above)
Comment (ec) [reply spec/api/v2/admin/users_spec.rb]: @gfedorenko no, we need to add additional entity.
Activities inside this entity used in several endpoints, so we cant remove them

**Comment by pkucherenk0** [comment]
@gfedorenko Pls add new calls to documentation.
Functional side looks fine by me.

**Comment by ec** [comment]
@pkucherenk0 documentation will be updated automatically (if kite will not die again)

**Review by ec — APPROVED** [review]

**Review by mod — APPROVED** [review]

**Pull request merged**